keyboards/wiredin/protagonist/pro_h/mcuconf.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2024 Gondolindrim <gondolindrim@acheronproject.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef STM32_PWM_USE_ADVANCED
+#define STM32_PWM_USE_ADVANCED TRUE
+
```

Comment:
```suggestion
```

Reply:
This is needed for the WS2812 complementary output. The PCB uses STM32F411 with WS2812 PWM driver on pin B15

Reply:
This is not actually needed after #22830 had been merged (it's in master now); the corresponding ChibiOS code for that option was removed even earlier.  Your branch seems to be based on some older version which does not have that change yet; you may want to rebase (and actually test your code on top of the recent QMK).